Gifski/TimeRemainingEstimator.swift

```diff
@@ -10,13 +10,21 @@ final class TimeRemainingEstimator {
 	/// Begin fade out when remaining time reaches this amount.
 	let fadeOutThreshold: TimeInterval = 1
 
-	var progress: Progress? {
+	weak var progress: Progress? {
```

Comment:
The progress instance was retained in so many places:
- By the conversion view controller
- Here
- In DockProgress (https://github.com/sindresorhus/DockProgress/commit/3365c2be401b47bf4ab4fe7aa0ffbf7d1a9658fc)